net-im/psi/psi-1.2.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="be bg ca cs de en eo es et fa fi fr he hu it ja kk mk nl pl pt pt_BR ru sk sl sr@latin sv sw uk ur_PK vi zh_CN zh_TW"
+PLOCALE_BACKUP="en"
+
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz
+	https://github.com/psi-im/psi-l10n/archive/1.2.tar.gz -> psi-l10n-${PV}.tar.gz"
+
+inherit eutils l10n multilib qmake-utils
+
+DESCRIPTION="Qt XMPP client"
+HOMEPAGE="http://psi-im.org/"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="aspell crypt dbus debug doc enchant +hunspell ssl xscreensaver
+whiteboarding webengine webkit"
+
+# qconf generates not quite compatible configure scripts
+QA_CONFIGURE_OPTIONS="--build=.* --host=.* --mandir=.* --infodir=.* --sysconfdir=.* --localstatedir=.*"
+
+REQUIRED_USE="
+	?? ( aspell enchant hunspell )
+	webengine? ( !webkit )
+"
+
+RDEPEND="
+	app-crypt/qca:2[qt5]
+	dev-qt/qtgui:5
+	dev-qt/qtxml:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtmultimedia:5
+	dev-qt/qtx11extras:5
+	net-dns/libidn
+	sys-libs/zlib[minizip]
+	aspell? ( app-text/aspell )
+	dbus? ( dev-qt/qtdbus:5 )
+	enchant? ( >=app-text/enchant-1.3.0 )
+	hunspell? ( app-text/hunspell )
+	webengine? ( >=dev-qt/qtwebengine-5.7:5[widgets] )
+	webkit? ( dev-qt/qtwebkit:5 )
+	whiteboarding? ( dev-qt/qtsvg:5 )
+	xscreensaver? ( x11-libs/libXScrnSaver )
+"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )
+"
+PDEPEND="
+	crypt? ( app-crypt/qca[gpg] )
+	ssl? ( app-crypt/qca:2[ssl] )
+"
+RESTRICT="test"
+
+pkg_setup() {
+	MY_PN=psi
```

Comment:
There is no reason to do this in pkg_setup (usually we declare those on top, after EAPI), but `MY_PN` is the same as `PN` anyway unless I'm missing something?